src/classes/IGiftBatch.cls

```diff
@@ -0,0 +1,13 @@
+public interface IGiftBatch {
```

Comment:
At what point do we expect to marry the GiftBatchTotals class with this interface since this interface should also include and define a `GiftBatchTotals` which makes the `Money` type internal to it?

Reply:
Looks like you answered my question in the comment at the top!